Tools/autotest/bisect-helper.py

```diff
@@ -277,7 +281,7 @@ def run(self):
 
             if code == self.exit_fail_code():
                 with open("/tmp/fail-counts", "a") as f:
-                    print("Failed on run %u" % (i+1,), file=f)
+                    print(f, "Failed on run %u" % (i+1,))
```

Comment:
f.write() maybe ?

Reply:
Gah.  Fixed, thanks.

It's potentially an issue that bisect helper needs to work across both py2 and py3 - and preferably the autotests.  Hmmm.